src/tests/encrypt-dry-run/expected-stdout

```diff
@@ -1,12 +1,12 @@
-Values to encrypt:
-C main/extractor/ex-generic-v2/books
-  parameters.#apiToken
-C main/extractor/ex-generic-v2/cars
-  parameters.carsTokens.[0].#token
-  parameters.carsTokens.[1].#token
-  parameters.carsTokens.[3].token.#special
-C main/extractor/keboola.ex-db-mysql/with-rows
-  parameters.db.#passwd
-R main/extractor/keboola.ex-db-mysql/with-rows/rows/payments
-  parameters.#paymentKey
+Plan for "encrypt" operation:
```

Comment:
Vystup je rovnaky ako pri inych prikazoch (v dalsom PR sa budu nahradzovat `tab` za `spaces`, kedze to niekde robi problemy, ... ale tu som to nechal este jednotne na `tab`).